gui/src/components/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import classNames from "classnames";
+import { withRouteData } from "react-static";
+
+const styles = theme => ({
+  style: {
+    backgroundColor: "#F8F8F8",
```

Comment:
I feel like these values for color & spacing should come from the MaterialUI theme. There is some example usage [here](https://github.com/smartcontractkit/chainlink/blob/master/gui/src/components/PaddedCard.js#L8) and if you'd like to read more about how it works there is a bunch of information on the [MaterialUI website](https://material-ui.com/customization/themes/#themes)